tests/test_onnxml_imputer_converter.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests onnxml Imputer converter
+"""
+import unittest
+import warnings
+
+import numpy as np
+import torch
+from sklearn.impute import SimpleImputer
+
+from hummingbird.ml._utils import onnx_ml_tools_installed, onnx_runtime_installed, lightgbm_installed
+from hummingbird.ml import convert
+
+if onnx_runtime_installed():
+    import onnxruntime as ort
+if onnx_ml_tools_installed():
+    from onnxmltools import convert_sklearn
+    from onnxmltools.convert.common.data_types import FloatTensorType as FloatTensorType_onnx
+
+
+class TestONNXImputer(unittest.TestCase):
+    def _test_imputer_converter(self, norm):
+        warnings.filterwarnings("ignore")
+        model = SimpleImputer(strategy="mean", fill_value="nan")
```

Comment:
Does the tests work only with this strategy? Can we have tests for other strategies? If it is easy to do.

Reply:
I'm not sure if that makes sense/works....the ONNX spec itself doesn't allow you specify. As above, we discussed that onnx uses "constant".  So...to create different SKL models with others should only make sense for tests...or is there a case that I'm missing?

Reply:
it _does_ work for both `model = SimpleImputer(strategy="mean", fill_value="nan")` and `model = SimpleImputer(strategy="constant")` because that field is simply ignored by the converter :)